Gemfile

```diff
@@ -84,6 +84,7 @@ group :development, :test do
   gem "pry"
```

Comment:
Can remove pry from the Gemfile (according to Gemfile.lock, pry is a dependency of pry-rails, so it will get loaded in automatically anyways).

I have never used pry-remote or pry-byebug (to the best of my knowledge). "git blame" says that @mbarnett added them.